### PR TITLE
Add unregistration

### PR DESCRIPTION
**Pull request by zvision**
- implemented unregistration
- renamed button text from "Submit" to "Register"
- added unit tests for unregistration

### CODE REVIEW

File: src/app/datasource/participant-data-source.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+  disconnect(collectionViewer: CollectionViewer): void {
+    this.participantSubject.complete();
+  }
+
+  connect(collectionViewer: CollectionViewer): Observable<Participant[] | readonly Participant[]> {
+    return this.participantSubject.asObservable();
+  }
+
+  loadParicipants(eventId: string) {
```
Comment (olamat) [inline src/app/datasource/participant-data-source.ts]: Spelling need to be fixed

File: src/app/service/registration.service.spec.ts [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+      );
+
+      const request = httpMock.expectOne(PARTICIPANT_URL + '/event/' + eventId);
+      request.flush(responseMessage, { status: 404, statusText: 'Not Found' });
+      httpMock.verify();
+    });
+  });
+
+  describe('method deleteRegistration', () => {
+    it('should return 204 No Content when delete registartion', () => {
```
Comment (olamat) [inline src/app/service/registration.service.spec.ts:118]: Spelling

File: src/app/datasource/participant-data-source.ts (hunk 1 above)
Comment (zvision) [reply src/app/datasource/participant-data-source.ts]: Fixed

File: src/app/service/registration.service.spec.ts (hunk 2 above)
Comment (zvision) [reply src/app/service/registration.service.spec.ts:118]: Fixed

**Review by olamat — APPROVED** [review]

**Pull request merged**